charts/dv-pod/templates/_helpers.tpl

```diff
@@ -147,3 +147,47 @@ Create comma-separated list of fallback beacon node endpoints
 {{- join "," .Values.charon.fallbackBeaconNodeEndpoints -}}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Validate validator client type
+*/}}
+{{- define "dv-pod.validateValidatorClientType" -}}
+{{- if .Values.validatorClient.enabled -}}
+{{- $validTypes := list "lighthouse" "lodestar" "teku" "prysm" "nimbus" -}}
+{{- $currentType := .Values.validatorClient.type -}}
+{{- if not (has $currentType $validTypes) -}}
+{{- fail (printf "ERROR: Invalid validator client type '%s'. Valid options are: %s" $currentType (join ", " $validTypes)) -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Derive network name from chainId
+Maps chainId to the corresponding network name for validator client configuration
```

Comment:
so are we not going to change the values.yml to be specifying a network and will stick to specifying a chainId? should we consider going the other way around? specifying name and we map to chainId?

Reply:
Branches got mixed up:
This is the original intent with a fix:
https://github.com/ObolNetwork/helm-charts/pull/146

But here I already have everything I originally wanted @OisinKyne 
Dropping the duplicate @OisinKyne